src/DataStructures/Tensor/Tensor.hpp

```diff
@@ -494,3 +495,93 @@ std::ostream& operator<<(std::ostream& os,
      << "\n     Multiplicity: " << x.multiplicity(i) << " Index: " << i;
   return os;
 }
+
+namespace MakeWithValueImpl_detail {
+/// \ingroup DataStructures
+/// \brief Returns a Tensor whose DataVectors are the same size as `input`, with
+/// each element equal to `value`.
+template <typename Symm, template <typename...> class IndexList,
+          typename... Indices>
+struct MakeWithValueImpl<Tensor<DataVector, Symm, IndexList<Indices...>>,
+                         DataVector, double> {
+  static SPECTRE_ALWAYS_INLINE Tensor<DataVector, Symm, IndexList<Indices...>>
+  apply(const DataVector& input, const double& value) {
+    return Tensor<DataVector, Symm, IndexList<Indices...>>(input.size(), value);
```

Comment:
Can you just do `{input.size(), value}`? This may be problematic for 2d vectors because `initializer_list` is annoying

Reply:
no, error: converting to Tensor ... from initializer list would use explicit constructor Tensor::Tensor(Args&& ...)